File: 5.1 - sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    h = get_int("Height: ")
    if h >= 1 and h <= 8:
        break

for i in range(h):
    i += 1
    print((" " * (h - i)) + ("#" * i) + "  " + ("#" * i))

File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    do
    {
        height = get_int("height: ");
    }
    while (height < 1 || height > 8);

    for (int i = 0; i < height; i++)
    {
        for (int j = 0; j < (2 * height + 2); j++)
        {
            if (j < height - 1 - i)
            {
                printf(" ");
            }
            else if (j >= height - 1 - i && j < height)
            {
                printf("#");
            }
            else if (j >= height && j < height + 2)
            {
                printf(" ");
            }
            else if (j >= height + 2 && j <= height + 2 + i)
            {
                printf("#");
            }
        }
        printf("\n");
    }

}